vnc/vnc.c

```diff
@@ -646,8 +646,8 @@ resize_client_from_layout(struct vnc *v,
 
             /* Dump some useful info, in case we get here when we don't
              * need to */
-            log_debug_screen_layout("OldLayout", layout);
-            log_debug_screen_layout("NewLayout", layout);
+            log_screen_layout(LOG_LEVEL_INFO, "OldLayout", layout);
+            log_screen_layout(LOG_LEVEL_INFO, "NewLayout", layout);
```

Comment:
I think this should be LOG_LEVEL_ERROR since this block is for an error, right?

Reply:
Always OldLayout == NewLayout. Either one should be client_layout?

Reply:
Good spot - thanks